Dockerfile

```diff
@@ -9,4 +9,4 @@ RUN apt-get update && apt-get upgrade -y && apt-get install -y git
 COPY Pipfile* /
 RUN pipenv install
 
-ENTRYPOINT ["pipenv", "run", "my_app"]
+ENTRYPOINT ["pipenv", "run", "cc-slips"]
```

Comment:
Totally optional suggestion: we're juggling `cc-slips`, `cc_slips`, `credit-card-slips` across the repo, maybe just simplify to `slips`?